WordPress/gutenberg
# Framework: Bump memize and rememo dependencies to latest

Related: https://github.com/WordPress/gutenberg/pull/3745#issuecomment-360632486

This pull request seeks to bump the versions of our memoize and rememo dependencies which respectively have received bug fixes resolving a case where incorrect cached values could be returned depending if the number of arguments passed varied between invocations. This is best seen in the test cases and code changes of https://github.com/aduth/memize/commit/3771d235cc9d7c261f9fb753dfe3ef2f94fe8cf5 and https://github.com/aduth/rememo/commit/aab2fd3f09c5cc4998ed40c3381094cc76b9a4c8 respectively.

__Testing instructions:__

There are no expected changes by this pull request in the master branch, but there should be no regressions in any memoized state selector behavior.

You may manually install the updated versions to the `add/block-children` branch (nesting, #3745) and verify that serialization of Columns block markup occurs correctly.
